### DISCUSSION TITLE
What are identities in Hubs Cloud?

### DISCUSSION THREAD

**Discussion by joone-org** [mozilla/hubs#3057] [Hubs Cloud Questions] [answered]
What are identities in Hubs Cloud?

**Comment by robinkwilson** [comment, marked as answer]
**Usually used for specific names so you can know who's account is who's.**
For example, my admin account on my Hubs Cloud has the identity "Robin personal gmail" so I can identify who that account is for and which email for that one. 

**It can also indicate different groups for access.** 
For example, I am an event organizer. We have **week 1** group + a **week 2** group. 
As the admin, I add accounts for all users. The week 1 group, I assign "Week 1" as the identity. Week 2 group "Week 2". I _deactivate_ Week 2 group's accounts. After week 1 ends, I _deactivate_ the accounts with Week 1 and _enable_ Week 2.

Learn how to _activate/deactivate_ accounts via the [Hubs Docs: Limiting User Access to your Hub](https://hubs.mozilla.com/docs/hubs-cloud-limiting-user-access.html#configure-your-hub-to-deny-users-without-accounts)

**Reply by robinkwilson** [reply]
Identities are only seen by Admins in the Admin Panel > Accounts menu.
